### DISCUSSION TITLE
Can't connect to FoxEss H3-10

### DISCUSSION THREAD

**Discussion by jaal2001** [nathanmarlor/foxess_modbus#538] [Support] [answered]
Hi, just got my H3-10 and trying to connect it via modbus. I double checked all installation manuals twice, but out of ideas:

2024-02-10 17:52:51.700 WARNING (MainThread) [custom_components.foxess_modbus.modbus_controller] tcp://192.168.6.10:502 247 - 5 failed poll attempts: now not connected. Last error: Error reading registers. Type: RegisterType.HOLDING; start: 44008; count: 5; slave: 247 from tcp://192.168.6.10:502: Exception Response(131, 3, IllegalAddress)

This is the only relevant thing I can see in the log, but I'm out of idea how to solve this.
Any hints?

Setup:
H3-10 -> Elfin W11 (is showing Received and Sent bytes in the status page) 

Thanks!

**Comment by canton7** [comment]
You're using the latest beta, I think? The betas are work-in-progress, and may not yet support all inverter models. Try using the latest stable release.

**Comment by jaal2001** [comment]
I'm using 1.9 (assumed that was a non beta?)

**Reply by canton7** [reply]
44008 is one of the remote control registers, and support for remote control is being developed in the 1.10.0-beta series. 1.9 will not attempt to read that register. Therefore, you must be using 1.10.0b1 or higher.

**Reply by jaal2001** [reply]
Using v1.10.0b6 still produces the same error message. Should I try any lower version?

And just in case: Are the registers readable during nighttime? Or do I need to wait until daylight?

**Reply by canton7** [reply]
Yes, you need to be on 1.9. The 1.10 beta series do not yet support the H3.

Nighttime makes no difference.

**Reply by jaal2001** [reply, marked as answer]
Thanks for your patience. Going back (explicit selecting 1.9.0 in HACS) seemed to do the trick. Now I can see all the values.

Thanks for your support!

**Comment by jaal2001** [comment]
However it seems to read at least something, as the HA device page shows:
"H3-10.0-E - AUX by FoxESS"
